src/current/_includes/common/telemetry-table.html

```diff
@@ -0,0 +1,936 @@
+                },
+                "lastErrorCode": "",
+                "failureCount": "0",
+                "usedFollowerRead": false
+            },
+            "aggregatedTs": "0001-01-01T00:00:00Z",
+            "aggregationInterval": "0s"
+        }
+    ]
+  </code>
+  </td>
+ </tr>
+ <tr>
+  <td>Build information</td>
+  <td>Build information for this CockroachDB binary.</td>
+  <td>
+   <ul>
+    <li>Go Version</li>
+    <li>Tag</li>
+    <li>Build Time</li>
+    <li>Revision</li>
+    <li>CGO Compiler</li>
+    <li>CGO Target Triple</li>
+    <li>Platform</li>
+    <li>Distribution</li>
+    <li>Build Type</li>
+    <li>Channel</li>
+    <li>Env Channel</li>
+    <li>Dependencies</li>
+   </ul>
+  </td>
+  <td>
+   <code>
+ "build": {
+      "goVersion": "go1.22.5 X:nocoverageredesign",
+      "tag": "v24.2.0-alpha.2-dev-2f0b0399a5c9656bdf8a417d0ab57e4eb1ae0f64",
+      "time": "2024/08/12 06:03:39",
+      "revision": "2f0b0399a5c9656bdf8a417d0ab57e4eb1ae0f64",
+      "cgoCompiler": "gcc 6.5.0",
+      "cgoTargetTriple": "x86_64-pc-linux-gnu",
+      "platform": "linux amd64",
+      "distribution": "CCL",
+      "type": "development",
+      "channel": "official-binary",
+      "envChannel": "unknown",
+      "enabledAssertions": false,
+      "dependencies": null
+    }
+</code>
+  </td>
+ </tr>
+ <tr>
+  <td>CPU information</td>
+  <td>Information about the CPU on the machine hosting CockroachDB.</td>
+  <td>
+   <ul>
+    <li>Number of CPUs</li>
+    <li>Sockets</li>
+    <li>Cores</li>
+    <li>CPU Model</li>
+    <li>CPU Speed</li>
+    <li>Features</li>
+    <ul>
+     <li>CPU feature flags for first CPU</li>
+    </ul>
+   </ul>
+  </td>
+  <td>
+   <code>
+"cpu": {
+        "numcpu": 16,
+        "sockets": 16,
+        "cores": 1,
+        "model": "Intel(R) Xeon(R) CPU @ 2.60GHz",
+        "mhz": 2600.032,
+        "features": [
+            "fpu",
+            "tsc",
+            "pse",
+            "sse4_2",
+            "aes",
+            "avx",
+            "avx2",
+            "rdseed",
+            "smap",
+            "hypervisor"
+        ]
+    }  
+</code>
+  </td>
+ </tr>
+ <tr>
+  <td>Memory information</td>
+  <td>Information about the memory on the machine hosting CockroachDB.</td>
+  <td>
+   <ul>
+    <li>Total Memory</li>
+    <li>Available Memory</li>
+   </ul>
+  </td>
+  <td>
+   <code>
+"mem": {
+        "total": "67413340264",
+        "available": "38576678952"
+      }
+  </code>
+  </td>
+ </tr>
+ <tr>
+  <td>Hardware information</td>
+  <td>Information about the hardware on the machine hosting CockroachDB.</td>
+  <td>
+   <ul>
+    <li>Virtualization Type</li>
+    <li>CPU Info</li>
+    <li>Memory Info</li>
+    <li>Load Average</li>
+    <li>Cloud Provider</li>
+    <li>Cloud Provider Instance Class</li>
+   </ul>
+  </td>
+  <td>
+   <code>
+"virtualization": ""
+"loadavg15": 7.10,
+"provider": "gcp",
+"instanceClass": "n2-standard-16"
+  </code>
+See also CPU Info and Memory Info
```

Comment:
Same comment as below on adding a `<br>`

```suggestion
See also: CPU Info and Memory Info
```

Reply:
✅  and ✅